docs/import/manage_disaster.js

```diff
@@ -550,13 +552,23 @@ function onSetDisaster() {
         processedCurrentDisasterStateAssets = true;
       }
     });
-    disasterAssets.get(currentDisaster).then((assets) => {
+    const disasterLambda = (assets) => {
       if (getDisaster() === currentDisaster &&
           !processedCurrentDisasterSelfAssets) {
         // Don't do anything unless this is still the right disaster.
         initializeDamageSelector(assets);
         processedCurrentDisasterSelfAssets = true;
       }
+    };
+    // Handle errors in disaster asset retrieval by just emptying out.
+    disasterAssets.get(currentDisaster).then(disasterLambda, (err) => {
+      if (err &&
```

Comment:
I think you might mean pre-damage :) This is what happens without your PR, since the disaster folder doesn't exist yet. We can delete this with your PR, or keep it if we think they might open this disaster in a new page when the EE folder creation hasn't succeeded yet. Since it doesn't actually cost any processing time, I'm ok with leaving it in.